.travis.yml

```diff
@@ -28,7 +28,7 @@ git:
 
 install:
   - >
-    export IRIS_TEST_DATA_REF="1696ac3a823a06b95f430670f285ee97671d2cf2";
+    export IRIS_TEST_DATA_REF="22166475999c4ba0f106b9214b71b667b8aee016";
```

Comment:
This will need changing again, since the SHA of master when merged is not the same as that in the merged branch :  _this_ commit exists only in your fork, not in the main scitools repo.
New commit is : https://github.com/SciTools/iris-test-data/commit/fffb9b14b9cb472c5eb2ebb7fd19acb7f6414a30

Reply:
Done